packages/passwordgeneratorwin/0.1.1/README.md

```diff
@@ -0,0 +1,22 @@
+# passwordgeneratorwin
+[Espanso](https://espanso.org) extension for generating passwords and copy it to cliboard. For now this extension is only supported on windows systems, beacause of the difference of path between `dos` an `unix` systems. If your are looking this extension for `linux` or `mac` go to [passwordgeneratorlinux](https://github.com/viera97/passwordgeneratorlinux).
```

Comment:
I think Windows will use forward slashes in paths - you could use that format for an OS-independent package.